Automattic/wp-calypso
# Reader Lists: add confirm dialog when removing items from a list

#### Changes proposed in this Pull Request

It's a touch too easy to remove a list item at the moment - this PR adds a confirmation dialog to the process.

<img width="694" alt="Screen Shot 2020-11-05 at 15 38 30" src="https://user-images.githubusercontent.com/17325/98190954-09293280-1f7d-11eb-9032-d4f92df2df9b.png">

#### Testing instructions



Visit an existing Reader list and view the items. Try and delete an item, and ensure you see the dialog in the screenshot above. Ensure that the item is removed when you click 'Remove'.
